jwql/instrument_monitors/nircam_monitors/claw_monitor.py

```diff
@@ -128,13 +137,21 @@ def make_background_plots(self, plot_type='bkg'):
             measured vs model trending.
         """
 
+        columns = ['filename', 'filter', 'pupil', 'detector', 'effexptm', 'expstart_mjd', 'entry_date', 'mean', 'median', 
+                   'stddev', 'frac_masked']  # , 'total_bkg']
+
         # Get all of the background data.
-        query = session.query(NIRCamClawStats.filename, NIRCamClawStats.filter, NIRCamClawStats.pupil, NIRCamClawStats.detector,
-                              NIRCamClawStats.effexptm, NIRCamClawStats.expstart_mjd, NIRCamClawStats.entry_date, NIRCamClawStats.mean,
-                              NIRCamClawStats.median, NIRCamClawStats.stddev, NIRCamClawStats.frac_masked, NIRCamClawStats.total_bkg).all()
-        df_orig = pd.DataFrame(query, columns=['filename', 'filter', 'pupil', 'detector', 'effexptm', 'expstart_mjd',
-                                               'entry_date', 'mean', 'median', 'stddev', 'frac_masked', 'total_bkg'])
-        df_orig = df_orig.drop_duplicates(subset='filename', keep="last")  # remove any duplicate filename entries, keep the most recent
+        background_data = NIRCamClawStats.objects.all().values(*columns)
+        df_orig = pd.DataFrame.from_records(background_data)
+        # remove any duplicate filename entries, keep the most recent
+        df_orig = df_orig.drop_duplicates(subset='filename', keep="last")
+
+        try:
```

Comment:
Where did this `try` block come from?

Reply:
It came from my trying to get the monitor to run. I had forgotten that the existing code in develop had the two new columns that don't actually exist yet, and so I had to either figure out how to get those values before they were used or I had to comment them out. I eventually decided that I probably couldn't get them, but forgot to remove the block I'd copied from lower down in the file in my attempt to see if I could figure out the background in time to divide by it.

Either way I've removed it now.

Reply:
Thank you @york-stsci just wanted to double check!